standalone/pherry/src/lib.rs

```diff
@@ -1045,13 +1061,15 @@ async fn bridge(
                 sync_with_cached_headers(
                     &pr,
                     &para_api,
+                    cache_client.as_ref(),
                     info.blocknum,
                     info.para_headernum,
                     cached_headers,
                     args.sync_blocks,
-                    args.get_changes_batch_size,
                 )
                 .await?;
+                sync_state.authory_set_state = None;
```

Comment:
Why do we clear it?

Reply:
If synced from the cache server, the `authory_set_state` may be outdated. clear it to trigger re-fetch from the chain once the cache is drained.